lib/codelog/command/bump.rb

```diff
@@ -6,16 +6,16 @@ class Bump
       VALID_VERSION_TYPES = ['major', 'minor', 'patch'].freeze
       INITIAL_RELEASE_VERSION = '0.0.0'.freeze
 
-      def self.run(version_type, release_date)
-        Codelog::Command::Bump.new.run version_type, release_date
+      def self.run(version_type, release_date, options)
+        Codelog::Command::Bump.new.run version_type, release_date, options
       end
 
-      def run(version_type, release_date)
+      def run(version_type, release_date, options)
         unless VALID_VERSION_TYPES.include?(version_type.downcase)
           abort(Codelog::Message::Error.invalid_version_type(version_type))
         end
 
-        Codelog::Command::Release.run(next_version(version_type), release_date)
+        Codelog::Command::Release.run(next_version(version_type), release_date, options)
```

Comment:
@Guilherme2112    
I think that the way this feature was implemented (passing the `options` hash all the way down the command chain) breaks the [Open-closed principle](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle) of pretty much every class in the gem. 

Maybe there is a way to implement this that doesn't require the change of interface of so many classes.

I recommend that you reflect on "Which (or how many) class *absolutely* has to know that we are in `preview` mode", and then try to contain the preview logic in only those classes, without propagating the changes down the line.